pom.xml

```diff
@@ -2414,7 +2414,7 @@
         <axis2.jibx.wso2.version>1.6.1.wso2v11</axis2.jibx.wso2.version>
         <axis2.jaxb.wso2.version>${axis2.wso2.version}</axis2.jaxb.wso2.version>
         <axis2-transports.version>2.0.0-wso2v42</axis2-transports.version>
-        <h2database.wso2.version>2.2.220.wso2v1</h2database.wso2.version>
+        <h2database.wso2.version>2.2.224.wso2v2</h2database.wso2.version>
```

Comment:
I think this h2 version is used for intg tests. Did you check from where are we packing the h2? If it is from kernel, let's bump all the versions together

Reply:
- Kernel h2 version is bumped with https://github.com/wso2/carbon-kernel/pull/3838

Reply:
- Carbon Business Process h2 version is bumped with https://github.com/wso2/carbon-business-process/pull/684